src/meta/WebAnimation.ts

```diff
@@ -24,7 +76,7 @@ export class WebAnimations extends Base {
 		const keyframeEffect = new KeyframeEffect(
 			node,
 			fx,
-			timing
+			timing as AnimationEffectTiming
```

Comment:
I think you could have left it how it was just with this cast?

Reply:
yup, most likely, will fall back. thanks